zha/application/platforms/light/__init__.py

```diff
@@ -710,7 +711,7 @@ def __init__(
         if self._color_cluster_handler:
             self._min_mireds: int = self._color_cluster_handler.min_mireds
             self._max_mireds: int = self._color_cluster_handler.max_mireds
-        effect_list = []
+        effect_list = [EFFECT_OFF]
```

Comment:
I believe changing this here might cause the effect option to always appear in HA now, even if the light does not actually support effects (other than off). (Haven't tested though.)

If the light does not support any effects, we do not want to expose `EFFECT_OFF`, correct? (Or do we? Does it matter? Does HA remove not show a single "off" effect in that case maybe?)
We should have a look at this before merging.

Reply:
Should be addressed with https://github.com/zigpy/zha/pull/99/commits/aa870ef62f908aef14cb1081dbc76fa71a361cd8
(again, not tested)

Reply:
Ok, I'm not actually sure that is needed, as `_supported_features` will not contain `EFFECT` already.
We may be able to revert that commit. It would also still return `EFFECT_OFF` in `effect`, even though that's not in the supported effects lists.

Reply:
Ok, it works fine with that commit reverted.